api_tests/lib_sdk/lnd_instance.ts

```diff
@@ -0,0 +1,172 @@
+import { ChildProcess, spawn } from "child_process";
+import * as fs from "fs";
+import { E2ETestActorConfig } from "../lib/config";
+import { fileExists } from "./utils";
+import { HarnessGlobal } from "../lib/util";
+import * as path from "path";
+import lnService from "ln-service";
+import { Logger } from "log4js";
+import { LogReader } from "../lib/log_reader";
+
+declare var global: HarnessGlobal;
+
+export class LndInstance {
+    private process: ChildProcess;
+    private lndDir: string;
+    private lndGrpc: any;
+
+    constructor(
+        private readonly logger: Logger,
+        private readonly logDir: string,
+        private readonly actorConfig: E2ETestActorConfig,
+        bitcoindDataDir: string
+    ) {
+        this.lndDir = this.logDir + "/lnd-" + this.actorConfig.name;
+        fs.mkdirSync(this.lndDir);
+
+        this.writeLogFile(bitcoindDataDir);
+    }
+
+    public async start() {
+        const bin = process.env.LND_BIN ? process.env.LND_BIN : "lnd";
+
+        if (global.verbose) {
```

Comment:
You have a logger accessible here, no need to use the legacy logging.

Reply:
Fixed as suggested.